frontend/views/pages/GroupDashboard.vue

```diff
@@ -107,10 +114,25 @@ export default ({
     humanDate,
     handleIncomeClick (e) {
       sbp('okTurtles.events/emit', OPEN_MODAL, 'IncomeDetails')
+
+      // firstly, make sure the listener is attached to 'GROUP_PROFILE_UPDATE' more than twice.
+      // e.g. user dismiss the modal without actually 'updating the income-details' and then click on the banner again.
+      sbp('okTurtles.events/off', GROUP_PROFILE_UPDATE, this.closeBanner)
+      sbp('okTurtles.events/once', GROUP_PROFILE_UPDATE, this.closeBanner)
```

Comment:
Instead of this, it makes more sense for there to be a single `/on` inside of either `beforeMount` or `mounted`, and a single corresponding `/off` in `beforeDestroy`.